### ISSUE TITLE
Ignore by Type and PropertyType in CompareResult

### ISSUE THREAD

**Issue by amellini** [StevenGilligan/AutoCompare#20]
It would be useful to Ignore property by type (ex Guid) and to have in the result the PorpertyType also.

**Comment by StevenGilligan** [comment]
Good idea and should be fairly easy to implement. I'll take a look into this once I figure out why the AppVeyor builds don't work anymore since last time everything was working fine.
